clean_cache/action.yml

```diff
@@ -0,0 +1,90 @@
+name: 'clean_cache'
+description: 'clean the cache'
+inputs:
+  s3cmd:
+    type: string
+    required: true
+    description: 'command line tool'
+  cache_endpoint:
+    type: string
+    required: true
+    description: 'secrets.CACHE_ENDPOINT for actions-cache'
+  cache_access_key:
+    type: string
+    required: true
+    description: 'secrets.CACHE_ACCESS_KEY for actions-cache'
+  cache_secret_key:
+    type: string
+    required: true
+    description: 'secrets.CACHE_SECRET_KEY  for actions-cache'
+  cache_objects:
+    type: string
+    required: true
+    description: 'objects to delete for cleanup'
+
+runs:
+  using: composite
+  steps:
+    - name: Install s3cmd and gh
+      if: ${{ inputs.install_ghcli == 'true' }}
```

Comment:
What does this line do? Do we need it?

Reply:
if condition is not required here as we are not giving install_ghcli as input parameter